erpnext/stock/doctype/material_request_item/material_request_item.json

```
@@ -403,11 +410,42 @@
    "fieldtype": "Data",
    "label": "Manufacturer Part Number",
    "read_only": 1
+  },
+  {
+   "fieldname": "section_break_label",
+   "fieldtype": "Section Break"
+  },
+  {
+   "fieldname": "label_quantity",
+   "fieldtype": "Data",
+   "label": "Label Quantity"
+  },
+  {
+   "fieldname": "jar_type",
+   "fieldtype": "Data",
+   "hidden": 1,
+   "label": "Jar Type"
+  },
+  {
+   "fieldname": "column_break_label",
+   "fieldtype": "Column Break"
+  },
+  {
+   "fieldname": "labels_needed",
+   "fieldtype": "Data",
+   "label": "Labels Needed"
```

Had a word with Arundhati, change this to:

```suggestion
   "label": "Labels Needed (Wrap, Front, Back, Top, Bag, etc.)"
```